rosbridge_library/test/capabilities/test_publish.py

```diff
@@ -62,7 +62,7 @@ def cb(msg):
 
         subscriber_qos = QoSProfile(
             depth=10,
-            durability=DurabilityPolicy.TRANSIENT_LOCAL,
+            durability=DurabilityPolicy.VOLATILE,
```

Comment:
Please leave the tests as-is unless you have a specific reason for changing them
```suggestion
            durability=DurabilityPolicy.TRANSIENT_LOCAL,
```